server/src/core/symbols/symbol.rs

```diff
@@ -1618,35 +1622,37 @@ impl Symbol {
         let typ = symbol.borrow().typ().clone();
         match typ {
             SymType::CLASS => {
-                for symbol in symbol.borrow().all_symbols() {
-                    let name = symbol.borrow().name().clone();
-                    if let Some(vec) = result.get_mut(&name) {
-                        vec.push((symbol, None));
-                    } else {
-                        result.insert(name.clone(), vec![(symbol, None)]);
+                // Skip current class symbols for super
+                if !(matches!(symbol_type, Some(EvaluationSymbolType::Super))){
+                    for symbol in symbol.borrow().all_symbols() {
+                        let name = symbol.borrow().name().clone();
+                        if let Some(vec) = result.get_mut(&name) {
+                            vec.push((symbol, None));
+                        } else {
+                            result.insert(name.clone(), vec![(symbol, None)]);
+                        }
                     }
                 }
                 let bases = symbol.borrow().as_class_sym().bases.clone();
                 for base in bases.iter() {
                     //no comodel as we will process only model in base class (overrided _name?)
-                    Symbol::all_members(&base, session, result, false, from_module.clone(), acc);
+                    Symbol::all_members(&base, session, result, false, from_module.clone(), acc, None);
```

Comment:
for bases, don't we need to skip duplicate symbols by name ?

It only makes sense to have duplicate names in the all members for co-models, right? 
to discuss